parse_publications.sh

```diff
@@ -0,0 +1,5 @@
+# #!/usr/bin/env bash
+
+
+python plugins/bibliography/bibparser.py
```

Comment:
This call does only give minimal output, I only got:

'run'  8311.67 ms

Maybe you could add some more informative information in the output.

Reply:
added extra output information